urdf2casadi/urdfparser.py

```diff
@@ -70,6 +70,54 @@ def get_joint_info(self, root, tip):
 
         return joint_list, actuated_names, upper, lower
 
+    def get_dynamics_limits(self, root, tip):
+        """Using an URDF to extract joint max effort and velocity"""
+
+        chain = self.robot_desc.get_chain(root, tip)
+        if self.robot_desc is None:
+            raise ValueError('Robot description not loaded from urdf')
+
+        max_effort = []
+        max_velocity = []
+
+        for item in chain:
+            if item in self.robot_desc.joint_map:
+                joint = self.robot_desc.joint_map[item]
+                if joint.type in self.actuated_types:
+                    if joint.limit is None:
+                        max_effort += [cs.inf]
+                        max_velocity += [cs.inf]
+                    else:
+                        max_effort += [joint.limit.effort]
+                        max_velocity += [joint.limit.velocity]
+        max_effort = [cs.inf if x==None else x for x in max_effort]
+        max_velocity = [cs.inf if x==None else x for x in max_velocity]
```

Comment:
```suggestion
        max_velocity = [cs.inf if x is None else x for x in max_velocity]
```
Comparisons to singletons should be done with "is" rather than equality.